src/features/user/BulkCodes/forms/IssueCodesForm.tsx

```diff
@@ -242,6 +254,23 @@ const IssueCodesForm = (): ReactElement | null => {
                 value={occasion}
                 label={t('bulkCodes:occasion')}
               />
+              {bulkMethod === 'import' && (
+                <FormControl fullWidth>
+                  <InputLabel id="demo-simple-select-label">
+                    Notification language
+                  </InputLabel>
+                  <Select
+                    labelId="demo-simple-select-label"
+                    id="demo-simple-select"
+                    value={notificationLang}
+                    label="Notification language"
+                    onChange={handleLocale}
+                  >
+                    <MenuItem value={'en'}>English</MenuItem>
+                    <MenuItem value={'de'}>Deutsch</MenuItem>
+                  </Select>
+                </FormControl>
```

Comment:
Could you structure this using the `<TextField select>` approach that's followed in other places (e.g. src/features/user/ManageProjects/components/BasicDetails.tsx). Note that it does not need to be within a `Controller` in this case.
```
	<TextField
		label={t('manageProjects:ecosystem')}
		variant="outlined"
		select
		onChange={onChange}
		value={value}
		onBlur={onBlur}
		error={errors.metadata?.ecosystem !== undefined}
		helperText={
			errors.metadata?.ecosystem !== undefined &&
			errors.metadata.ecosystem.message
		}
	>
		{ecosystemTypes.map((ecosystem) => (
			<MenuItem key={ecosystem} value={ecosystem}>
				{t(`manageProjects:ecosystemTypes.${ecosystem}`)}
			</MenuItem>
		))}
	</TextField>
```